api/src/main/java/io/opentelemetry/api/trace/EndSpanOptions.java

```diff
@@ -15,22 +15,21 @@
 @Immutable
 @AutoValue
 public abstract class EndSpanOptions {
-  private static final EndSpanOptions DEFAULT = builder().build();
+  private static final EndSpanOptions DEFAULT = createWithEndTimestamp(0L);
+
+  /**
+   * Returns a {@link EndSpanOptions} indicating the span ended at the given {@code endTimestamp},
+   * in nanoseconds.
+   */
+  public static EndSpanOptions createWithEndTimestamp(long endTimestamp) {
```

Comment:
maybe just `create(long endTimestamp)` ?

Reply:
Renamed - I think either this or removing the class and inlining into `Span` are both good options.